### DISCUSSION TITLE
NextJs + custom ExpressJs server (typescript)

### DISCUSSION THREAD

**Discussion by stychu** [privatenumber/pkgroll#59] [Q&A]
### Question

Hi there, Im researching how to build a NextJs project with custom Express server. According to the docs and If I understand correctly I should be able to use `pkgroll` to be able to build my custom express server written in typescript and serve it as just plain js.

My project contains:

```
/src - (NextJS app)
/root-server - (custom express server)
```

The `/src` is being build by next `next build` where the `/root-server` should be build by pkgroll. Could you point me how to configure  pkgroll to `/root-server` and output build to the `./dist/root-server/...`

Ideally I would have a configuration where next app is build by `next build`, server is build by `pkgrool` and dev env is run in watch mode by `tsx`

**Comment by privatenumber** [comment]
Try doing `pkgroll --help`. There should be flags to change the src and dist directories. Examples in tests: https://github.com/privatenumber/pkgroll/blob/develop/tests/specs/builds/src-dist.ts
